docs/flashing_bootloadhid.md

```diff
@@ -0,0 +1,70 @@
+# BootloadHID Flashing Instructions and Bootloader Information
+
+ps2avr(GB) boards use an atmega32a microcontroller and a different bootloader. It is not flashable using the regular QMK methods.
+
+General flashing sequence:
+
+1. Enter the bootloader using any of the following methods:
+    * Tap the `RESET` keycode (may not work on all devices)
+    * Hold the salt key while plugging the keyboard in (usually documented within keyboard readme) 
+2. Wait for the OS to detect the device
+3. Flash a .hex file
+4. Reset the device into application mode (may be done automatically)
+
+## bootloadHID Flashing Target
+
+Using the qmk installation script, detailed [here](newbs_getting_started.md), the required bootloadHID tools should be automatically installed.
+
+To flash via the command line, use the target `:bootloadHID` by execute the following command:
+
+    make <keyboard>:<keymap>:bootloadHID
+
+## GUI Flashing
+
+### Windows
+1. Download [HIDBootFlash](http://vusb.wikidot.com/project:hidbootflash).
+2. Place your keyboard into reset. 
+3. Ensure the configured VendorID is `16c0` and ProductID is `05df`
```

Comment:
```suggestion
3. Ensure the configured USB Vendor ID is `16C0` and Product ID is `05DF`.
```

Reply:
The terminology used was intended to match the applications usage,
![image](https://user-images.githubusercontent.com/16520359/60147487-78a5e400-97c5-11e9-8ff9-3111e1202070.png). Would bold/italic help in this instance?

Reply:
Should be easy enough to grok, but I removed the USB part just in case.